pkg/controller/populators/populator-base.go

```diff
@@ -199,6 +199,7 @@ func (r *ReconcilerBase) createPVCPrime(pvc *corev1.PersistentVolumeClaim, sourc
 			Resources:        pvc.Spec.Resources,
 			StorageClassName: pvc.Spec.StorageClassName,
 			VolumeMode:       pvc.Spec.VolumeMode,
+			Selector:         pvc.Spec.Selector,
```

Comment:
Confirming we are already good on populator clone flows?

Reply:
Good point, I haven't checked

Reply:
Yeah looks fine. The temp restore PVC for snapshot cloning will not use the selector but that's not the target PVC, just in intermediary